build.sh

```diff
@@ -11,22 +11,11 @@ $MiniCover reset
 dotnet test --no-build test/MiniCover.XUnit.Tests/MiniCover.XUnit.Tests.csproj
 dotnet test --no-build test/MiniCover.NUnit.Tests/MiniCover.NUnit.Tests.csproj
 $MiniCover uninstrument
-$MiniCover htmlreport --threshold 90 
-$MiniCover xmlreport --threshold 90
 $MiniCover report --threshold 90
-$MiniCover opencoverreport --threshold 90
-
-$MiniCover instrument --sources "test/**/*.cs" --assemblies "test/**/bin/**/*.dll" --with-hiting-tests
```

Comment:
I think we can remove the opt-in to collect tests and make it default.
After the performance improvements, the only impact on having it enabled by default is the big hits file that is generated, but I tackled that issue on this PR.